etc/config-template.toml

```diff
@@ -11,17 +11,33 @@
 # log-file = ""
 
 [readpool.storage]
-# size of thread pool for high-priority read operations
+# size of thread pool for high-priority operations
```

Comment:
Storage readpool usually is used by kv get/batch_get/scan, there are lighter than coprocessor's read request, so i think the storage readpool size should small than coprocessor readpool in default.

Reply:
got

Reply:
Let's keep it here. The change is very small that it won't bother your review :)